modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqTranslate.scala

```diff
@@ -106,7 +135,19 @@ class SeqTranslate(site: Site, systems: Systems, settings: Settings) {
     val instName = configs.headOption.map(extractInstrumentName).getOrElse("Unknown instrument")
 
     steps match {
-      case (errs, ss) => (errs, if(ss.isEmpty) None else Some(Sequence[Action](obsId.stringValue(), SequenceMetadata(instName, None, name), ss)))
+      case (errs, ss) => (
+        errs,
+        if (ss.isEmpty)
```

Comment:
I had a commit where I used `option` instead of the `if...else` but removed it at the last moment because with that specific commit nothing was getting loaded. I didn't get into details why it was happening.